commons/build.gradle

```diff
@@ -25,4 +25,6 @@ dependencies {
     // For now taking the version from kafka-clients
     compileOnly "com.github.luben:zstd-jni:$zstdVersion"
     testImplementation "com.github.luben:zstd-jni:$zstdVersion"
+    implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
```

Comment:
Common formatting would be good + perhaps to the root gradle file since used in both modules?

Reply:
Well, all `build.gradle` could use some cleaning anyway. Made the common format